frontend/src/NavBar.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import AppBar from '@material-ui/core/AppBar';
+import Toolbar from '@material-ui/core/Toolbar';
+import Typography from '@material-ui/core/Typography';
+import Button from '@material-ui/core/Button';
+
+const useStyles = makeStyles((theme) => ({
```

Comment:
ooc, is there a way to move css/styling code to `.css` file separate from the `.js` files?

Reply:
as far as I know this is how the material UI framework does styling (with React hooks)
https://material-ui.com/styles/basics/#hook-api

Reply:
ok I think there is a way to move the styling code into another file. but if it is not a big part of the code itself, I think it would be okay to keep in the same js file

Reply:
yup, if you think it's okay, then it should be fine. as long as we don't have huge chunk of css in the js code!